Source/WebCore/platform/graphics/coreimage/FEComponentTransferCoreImageApplier.mm

```diff
@@ -67,6 +68,49 @@
     if (!inputImage)
         return false;
 
+    bool needsGamma = m_effect->redFunction().type == ComponentTransferType::FECOMPONENTTRANSFER_TYPE_GAMMA
+        || m_effect->greenFunction().type == ComponentTransferType::FECOMPONENTTRANSFER_TYPE_GAMMA
+        || m_effect->blueFunction().type == ComponentTransferType::FECOMPONENTTRANSFER_TYPE_GAMMA;
+
+    if (needsGamma) {
+        NSError *error = nil;
+        NSArray<CIKernel *> *kernels = [CIKernel kernelsWithMetalString:@R"(
+            [[stitchable]] float4 gammaFilter(coreimage::sampler src, float ampR, float expR, float offR,
```

Comment:
is `[[stitchable]]` required here?

Reply:
```
Kernel creation failed: Error Domain=CIKernel Code=12 "(null)" UserInfo={CINonLocalizedDescriptionKey=Cannot find a valid stitchable Metal function in the source}
```
yes, it seems so. maybe compiling with MTLLibrary data will make it unnecessary?